assets/js/partials/cookies.js

```diff
@@ -32,6 +34,6 @@ $(document).ready(function() {
   $('#hidecookie').click(function () {
     $('.cookie-block').slideUp('slow');
     $('footer').css('margin-bottom', '0');
-    createCookie('greenpeace', 'policy-accepted');
+    createCookie('greenpeace', '1', 365);
```

Comment:
should the value here be '2' ?
On firefox I am getting the iframe and the 'content is blocked' message at the same time

Reply:
![screenshot-2018-5-24 development planet 4 website](https://user-images.githubusercontent.com/14092452/40481777-19eb3256-5f5b-11e8-984c-df8419b71acf.png)